WcaOnRails/app/views/competitions/_index_table.html.erb

```diff
@@ -1,10 +1,25 @@
 <ul class="list-group">
   <li class="list-group-item"><strong><%= title %></strong></li>
   <% competitions.each_with_index do |competition, index| %>
+    <% if index > 0 && competition.year != competitions[index - 1].year %>
+      <li class="list-group-item break"><%= competition.year %></li>
+    <% end %>
     <li class="list-group-item <%= competition.is_over? ? "past" : "not-past" %>">
-      <span class="date"><%= date_range(competition.start_date, competition.end_date, separator: '-', show_year: false) %>, <%= competition.year %></span>
+      <span class="date">
+        <% if competition.is_over? %>
+        <%=  icon('hourglass-end') %>
+        <% elsif competition.in_progress? %>
+        <%=  icon('hourglass-half') %>
+        <% else %>
+        <%=  icon('hourglass-start') %>
+        <% end %>
+        <%= date_range(competition.start_date, competition.end_date, separator: '-', show_year: false) %>, <%= competition.year %>
+      </span>
       <span class="competition-info">
-        <p class="competition-link"><%= link_to competition.name, competition_path(competition) %></p>
+        <p class="competition-link">
+          <i class="flag f16 <%= competition.country.iso2.downcase %>"></i>
```

Comment:
What happenes here for competitions like [FMC Europe 2016](https://www.worldcubeassociation.org/results/c.php?i=FMCEurope2016)? Here's what it looks like in our production database:

```sql
mysql> select id, countryId from Competitions where id='FMCEurope2016';
+---------------+-----------+
| id            | countryId |
+---------------+-----------+
| FMCEurope2016 | XE        |
+---------------+-----------+
1 row in set (0.00 sec)

mysql> 
```

See our other screwy countries here: https://github.com/cubing/worldcubeassociation.org/blob/131bb018854e23a7f8e74131280d88996c763585/WcaOnRails/app/models/country.rb#L1808-L1833. Perhaps we could come up with some cute multiple flag icon for these guys?